ais/local/settings.py

```diff
@@ -34,14 +34,15 @@
         'NAME': 'ais_dev',
         'USER': 'ais_dev',
         'PASSWORD': 'ais_dev',
-        'HOST': '127.0.0.1', # For Vagrant
-        # 'HOST': 'db',  # For docker
+        # 'HOST': '127.0.0.1', # For Vagrant
+        'HOST': 'db',  # For docker
     }
 }
 
 SALES_HOOK_URL = 'https://hooks.slack.com/services/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'
 RECRUITMENT_HOOK_URL = 'https://hooks.slack.com/services/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'
 
+
```

Comment:
Remove it, no need to commit settings if you are not implementing critical information.